opentelemetry-sdk/CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## vNext
 
-- *Breaking* Make `force_flush()` in `PushMetricExporter` synchronous
+- *Breaking*: Calls to `MeterProviderBuilder::with_resource`, `TracerProviderBuilder::with_resource`, 
```

Comment:
```suggestion
- Calls to `MeterProviderBuilder::with_resource`, `TracerProviderBuilder::with_resource`, 
```

Reply:
My suggestion is to just mention the change, and not call it "breaking" to avoid scaring people unnecessarily!